adal/src/androidTest/java/com/microsoft/aad/adal/BrokerProxyTests.java

```diff
@@ -1170,13 +1170,11 @@ private Context getMockContext(final Signature signature, final String brokerPac
         return mockContext;
     }
 
-    @SuppressLint("PackageManagerGetSignatures")
     private PackageManager getPackageManager(final Signature signature, final String packageName,
                                              boolean permissionStatus) throws NameNotFoundException {
         PackageManager mockPackage = mock(PackageManager.class);
-        PackageInfo info = Util.addSignatures(new PackageInfo(), new Signature[]{signature});
-
-        when(mockPackage.getPackageInfo(packageName, PackageHelper.getPackageManagerFlag())).thenReturn(info);
+        final PackageInfo mockedPackageInfo = new MockedPackageInfo(new Signature[]{signature});
```

Comment:
Can we keep Util.addSignature, but have it return a mockito-ed MockedPackageInfo instead?

Reply:
I personally don't see the need for it as `MockedPackageInfo `wraps everything up nicely